File: cpp/cezar.cpp
#include <iostream>
using namespace std;

int main()
{
    
    int klucz = 3;  // klucz - o ile będziemy przesuwać litery w alfabecie
    
    string napis;   //napis który szyfrujemy (WIELKIE LITERY)
    cout<<"Podaj napis: ";
    cin>>napis;
    
    //pętla przechodząca po każdej literce napisu
    int i = 0;
    while (i<napis.size()){
        
        if (char(napis[i] + klucz) > 'Z'){      //sprawdzamy czy szyfr nie wychodzi poza alfabet (np. z + 3 = c)
            cout<<char(napis[i] + klucz - 26);  //zapetlamy alfabet czyli cofamy sie na pocztaek (alfabet ma 26 znakow)
        }else{
            cout<< char(napis[i] + klucz) ;     // przesuwamy litere o "klucz" miejsc
        }
         
        i++; // licznik pętli
    }
    
    
    
    
    return 0;
}

File: python/cezar.py
# print(ord('A'))
# print(chr(65))

napis = input("Podaj napis")

i= 0
while i < len(napis):
    
    if chr(ord(napis[i]) + 3) > 'Z':
        print(chr(ord(napis[i]) + 3 -26))
    else:
        print(chr(ord(napis[i]) + 3))
        
    i+=1
